scripts/release/theme-extractor.mjs

```diff
@@ -2,13 +2,12 @@
 import fs from 'node:fs';
 import path from 'node:path';
 import fg from 'fast-glob';
+import yargs from 'yargs/yargs';
+import { hideBin } from 'yargs/helpers';
 
 import { log, errorHandler, success, ROOT, getJSON } from '../helpers/esm.mjs';
 import { ELEMENT_DIST, PACKAGE_ROOT, getElementList, getElementTagName } from './util.cjs';
 
-// List of themes to be extracted
-const THEMES = ['halo', 'solar'];
-
 // Element package scope
 const PACKAGE_NAME = (await getJSON(`${PACKAGE_ROOT}/package.json`, import.meta)).name;
```

Comment:
This is the reason why theme extractor is currently giving wrong output, 

`import '@refinitiv-ui/halo-theme/icon/themes/halo/dark';`

instead of 

`import '@refinitiv-ui/elements/icon/themes/halo/dark';`

because it reads package name from `package.json` that it's executing. We could hard code this value as elements as it is not going to change so often.